test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +36,35 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void elementsHaveNull() throws Exception {
+        // empty list
+        assertNoneNull();
+
+        // has null
+        assertAnyNull((Object) null);
+        assertAnyNull(null, "CS2103");
+        assertAnyNull(null, null);
+
+        // none null
+        assertNoneNull("5");
+        assertNoneNull("CS2103", 5);
+        assertNoneNull(2103);
```

Comment:
Could add a few more test cases with:
- Multiple (>2) items
- Combinations of multiple null and non-null items